src/Crypt32.Shared/Crypt32.cs

```diff
@@ -51,5 +51,12 @@ public static partial class Crypt32
         [DllImport(nameof(Crypt32), SetLastError = true)]
         [return: MarshalAs(UnmanagedType.Bool)]
         private static extern bool CertCloseStore(IntPtr hCertStore, CertCloseStoreFlags dwFlags = CertCloseStoreFlags.None);
+
+        [DllImport(nameof(Crypt32), CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern bool CertGetCertificateContextProperty(
+                IntPtr pCertContext,
+                CertPropId dwPropId,
+                IntPtr pvData,
```

Comment:
`pvData` should be typed as `void*`, and `pcbData` should be `int`. Please review [parameter type guidelines](https://github.com/AArnott/pinvoke/blob/master/CONTRIBUTING.md#method-parameter-types)